packages/spin/spin.0.8.3/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+synopsis: "OCaml project generator"
+description: "OCaml project generator"
```

Comment:
Note for next time: only `synopsis` is mendatory. Having both is redundant as if `description` is not given, `description = synopsis`. You might want to elaborate a little bit more on the description if you want.